justfile

```diff
@@ -1,25 +1,18 @@
 # list all tasks
 default:
-    @just --list --unsorted
+    @just --list
 
 # run tests
 test *FLAGS:
-    #! /usr/bin/env bash
-    uv run pytest tests -m "not integration" {{ FLAGS }}
-    if ! git diff --quiet pyproject.toml; then
-      uv run pytest .github/workflows {{ FLAGS }}
-    fi
+    @uv run pytest .github/workflows tests -m "not integration" {{ FLAGS }}
```

Comment:
isn't this skipping the tests?

Reply:
do you mean the `@` prefix? 

the varargs just runs it on 2 folders instead of 1 now (we do skip exchange tests, but that was always the case). it didn't make any sense why i'd only run unit tests if the project toml changed 

```
$ just test
========= 197 passed in 6.78s ==========
```

Reply:
ah ok - I thought removing `uv run pytest tests -m "not integration" {{ FLAGS }}` means it would only run workflow tests?

Reply:
ah i see, there's 2 folders there that isn't super obvious. added trailing slashes for clarity if that makes more sense 3309b70